clang/lib/DPCT/CMakeCommands.inc

```diff
@@ -0,0 +1,110 @@
+//===--------------- CMakeCommands.inc ------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+/****************************************************************************
+*
+* ENTRY_TYPE(Command, ProcessedOrNot, MigrationDesc)
+*  Command: CMake Command Name
+*
+*  ProcessedOrNot:  false|true
+*    false: this Command has not been processed.
+*    true:  this Command has been processed, need check MigrationDesc field to
+*           see if it has be migrated Commently.
+*
+*  MigrationDesc: Comment[: Comment] | Unsupported: Comment
+*    Comment:   this Command has been migrated.
+*    Unsupported:  this Command can not be migrated currently.
+*                  Must add Comment to explain why this API is Unsupported.
+*    Comment:      extra comment on the migration.
+*
+****************************************************************************/
+ENTRY_TYPE(cmake_minimum_required, true, "comment")
```

Comment:
should we add one more field to indicate, it is CUDA-specific?   
if it is CUDA-specific and it's false on support, then it emits a warning message.

Reply:
Good advice; it makes the inc file more friendly to read; I will add them.